### ISSUE TITLE
Issues when running buildbreaker notebook

### ISSUE THREAD

**Issue by fridex** [thoth-station/notebooks#36]
I installed missing dependencies - jupyter-require, jupyter-tools and spicy. The first two were installed by un-commenting the first cell, the latter one had to be installed manually (it can be added to dependency listing to the jupyter image).

After installation the notebook was stuck with message "waiting for kernel". I saw the following lines in jupyterhub console log:

```
[W 16:36:21.379 NotebookApp] Replacing stale connection: 42d2f4c9-63c8-47d3-9cae-1def684ac5d6:cc8ba9783c314aac87d7b09c297c1dd7
[W 16:36:21.686 NotebookApp] 404 GET /static/pdfmake.js?v=20190401163513 (::1) 4.45ms referer=http://localhost:8888/notebooks/development/Build%20Breaker%20identification%20-%20PIP.ipynb
[W 16:36:21.693 NotebookApp] 404 GET /static/vfsfonts.js?v=20190401163513 (::1) 4.84ms referer=http://localhost:8888/notebooks/development/Build%20Breaker%20identification%20-%20PIP.ipynb
[W 16:36:21.697 NotebookApp] 404 GET /static/datatables.net.js?v=20190401163513 (::1) 7.28ms referer=http://localhost:8888/notebooks/development/Build%20Breaker%20identification%20-%20PIP.ipynb
```

I was basically in this state, I couldn't run the notebook, it still waited for the kernel to become ready.

I tried to reinstall a fresh virtual environment by removing virtualenv and installing all the dependencies. Now I got the following error when running the notebook:

```python
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-7-222c654363c5> in <module>
      1 
      2 
----> 3 from jupyter_tools import utils
      4 
      5 utils.install_nbextension('jupyter_require')

ModuleNotFoundError: No module named 'jupyter_tools'
```

I've spotted the following message in the installation log:

```
jupyter-tools 0.1.0 has requirement jupyter-require==0.1.13, but you'll have jupyter-require 0.2.0rc0 which is incompatible.

jupyter-tools 0.1.0 has requirement jupyter-require==0.1.13, but you'll have jupyter-require 0.2.0rc0 which is incompatible.
Installing collected packages: jupyter-require
  Found existing installation: jupyter-require 0.1.13
    Uninstalling jupyter-require-0.1.13:
      Successfully uninstalled jupyter-require-0.1.13
Successfully installed jupyter-require-0.2.0rc0
```

**Comment by CermakM** [comment]
Thanks for testing this out! There were indeed some issues and missing spacy installation commands.

There were some leftovers in cell and notebook metadata, I deleted them and fixed requirements on the jupyter-tools side. Both jupyter-tools and jupyter-require are changing quite quickly and from time to time I introduce breaking changes, sorry for that... It should be fixed by #37.

Please, check once again. Thanks!

**Comment by fridex** [comment]
Thanks for the fix, I can open the notebook now.
